CRM/Contact/Page/View/UserDashBoard.php

```diff
@@ -64,7 +64,7 @@ public function __construct() {
       CRM_Utils_System::redirect(CRM_Utils_System::url('civicrm/dashboard', 'reset=1'));
     }
 
-    $this->_contactId = CRM_Utils_Request::retrieve('id', 'Positive', $this);
+    $this->_contactId = CRM_Utils_Request::retrieve('cid', 'Positive', $this);
```

Comment:
It actually depends on how user dashboard is loaded. This worked fine when url was loaded as `civicrm/user?reset=1&id=203`. As `cid` is most common param for contact id, user dashboard is also updated to use this as an identifier.